runners/genlayer-py-std/src/genlayer/py/evm/__init__.py

```
@@ -49,3 +50,11 @@
 from .calldata import *
 from .support import *
 from .generate import contract_generator, ContractProxy, ContractDeclaration
+
+import typing
+from ..types import Address
+
+
+def contract_interface[TView, TWrite](
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (genlayer.py.evm.__init__, line 58)'

(E0001)